tls-attestation/draft-fossati-tls-attestation.md

```diff
@@ -57,40 +57,17 @@ author:
 normative:
   RFC2119:
   RFC8446:
+  attestation-wrapper:
+    target: https://link.example.com
+    title: Attestation Wrapper
+    author:
+      -
+        org: Fossati/Arm
+    date: October 2022
```

Comment:
I'm trying to get this published shortly.  Will update it as soon as I get the ref from the IETF Datatracker.